### PR TITLE
Action readucers

### PR DESCRIPTION
**Pull request by Kwamay**
In this model, I...........
-[ ] Configured a Redux store

-[ ] Defined state of books that adds and remove reducer books

-[ ] Incuded a reducer that checks the status of reducer category

### CODE REVIEW

File: src/redux/categories/categoriesSlice.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+const categoryReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_CATEGORY:
+      return action.text;
+    default: return state;
+  }
+};
+
+export default categoryReducer;
```
Comment (OyePriscilla) [inline src/redux/categories/categoriesSlice.js]: - [ ] Please as stated in the requirement, kindly use Redux Toolkit to create a slice for managing the state of `categories` in your Redux store. This [tutorial might be of help](https://redux-toolkit.js.org/tutorials/quick-start#create-a-redux-state-slice). 

Untitled

File: src/redux/books/booksSlice.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+export const addnewBook = (id, title, author) => ({
+  type: ADD_BOOK,
+  content: {
+    id,
+    title,
+    author,
+  },
+});
+
+export const removeBook = (id) => ({
```
Comment (OyePriscilla) [inline src/redux/books/booksSlice.js]: - [ ] Please as stated in the requirement, kindly use Redux Toolkit to create a slice for managing a list of books in your Redux store. This [tutorial might be of help](https://redux-toolkit.js.org/tutorials/quick-start#create-a-redux-state-slice). 

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Kwamay 👍 👍 👍 👍, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
`@reduxjs/toolkit` & `react-redux` present in `package.json`. ✔️ 
Structure of the files follow the `ducks pattern` ✔️ 
Configured a Redux store ✔️ 

## Required Changes ♻️
- [ ] Please kindly remove these commented code, this will make your codebase to be clean and more professional. 👍 

Untitled

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bellom** [comment]
Dear Senior, 

Seems like a GH glitch, this commit 94c3e31 came in after the first CR review.

image

**Review by fobadara — APPROVED** [review]
Hi @Kwamay, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

APPRVED
## Highlights
✅ Descriptive README
✅ Descriptive PR summary
✅ Clean code


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**